app/views/users/show.html.erb

```diff
@@ -8,4 +8,13 @@
       </h1>
     </section>
   </aside>
+  <div class="col-md-8">
+    <% if @user.microposts.any? %>
+      <h3><%= t "post.view.count_post", count: @user.microposts.count %></h3>
```

Comment:
so sánh length, size, count giúp e

Reply:
1. .count sử dụng câu lệnh SELECT COUNT(*) FROM để truy vấn số lượng bản ghi
2. .lenght nó sẽ đếm số lượng mà không sử dụng thêm câu truy vấn nếu đối tượng đó đã được nạp, ngược lại, nó sẽ nạp toàn bộ object của bạn vào bộ nhớ và thực hiện đếm 
3. .size ,khi object đã đuợc load, nó sẽ thực hiện trên chính object đó trong bộ nhớ (không query vào database, tuơng tự lenght ở lần thứ 2 trở đi), còn khi object đó chưa được load, nó sẽ thực hiện câu lệnh (SELECT COUNT(*) FROM…) giống như .count